app/gather/assets/css/_survey-view.scss

```diff
@@ -99,18 +100,49 @@
   display: flex;
   flex-wrap: wrap;
   align-items: baseline;
+
+  .tab {
+    padding: 1.4rem 0 1.1rem;
+    height: $toolbar-height;
+  }
+
+  li:first-child .tab {
+    padding-left: 5vw;
+  }
+
+  .dashboard-tab + li .tab {
+    margin-left: 1.2rem;
+  }
+
+  .dashboard-tab {
+    background: rgba($white, .2);
+    border-right: 1px solid $white;
+
+    .tab {
+      margin-right: 1.2rem;
+    }
+  }
 }
 
 .toolbar-filter {
   margin-left: auto;
+  padding-right: 5vw;
+  .alert {
+    background: none;
+    color: white;
+    font-size: $font-size-s;
+  }
+  .loading-spinner {
+    border-top-color: lighten($action-color, 30);
+    border-right-color: lighten($action-color, 30);
+  }
```

Comment:
Lint warnings

```text
./css/_survey-view.scss
  130:1   warning  Space expected between blocks                                                             empty-line-between-blocks
  132:12  warning  Color 'white' should be written in its hexadecimal form #ffffff                           no-color-keywords
  135:1   warning  Space expected between blocks                                                             empty-line-between-blocks
  411:16  warning  Property `padding` should be written more concisely as `1rem 0` instead of `1rem 0 1rem`  shorthand-values

./css/base/_global.scss
  195:1  warning  Space expected between blocks  empty-line-between-blocks
```